test/update.js

```diff
@@ -191,8 +191,8 @@ test('with modules', async t => {
           type: 'content',
           subtype: 'Q17737',
           main: '',
-          authors: [`dat://${encode(profileKey)}`],
-          parents: [`dat://${encode(parent1Key)}`]
+          authors: [`hyper://${encode(profileKey)}`],
+          parents: [`hyper://${encode(parent1Key)}`]
```

Comment:
```suggestion
          authors: [`${encode(profileKey)}`],
          parents: [`${encode(parent1Key)}`]
```

See https://github.com/p2pcommons/sdk-js/issues/179